coops-ui/tests/unit/ARDate.spec.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint promise/param-names: 0, prefer-promise-reject-errors: 0 */
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import Vuelidate from 'vuelidate'
+
+import store from '@/store/store'
+import { shallowMount } from '@vue/test-utils'
+import ARDate from '@/components/AnnualReport/BCorp/ARDate.vue'
+
+Vue.use(Vuetify)
+Vue.use(Vuelidate)
+
+describe('AnnualReport - Part 1 - UI', () => {
+  beforeEach(() => {
+    // init store
+    store.state.nextARDate = '2020-09-18T23:15:53.785045+00:00'
+  })
+
+  it('initializes the store variables properly', () => {
+    const wrapper = shallowMount(ARDate, { store })
+    const vm: any = wrapper.vm
+
+    expect(vm.$store.state.nextARDate).toEqual('2020-09-18T23:15:53.785045+00:00')
+    wrapper.destroy()
+  })
+
+  it('succeeds when the Annual report date outputs are correct', () => {
+    const wrapper = shallowMount(ARDate, { store })
+    const vm: any = wrapper.vm
+    const regex = / (?!.* )/
+    const today = new Date().toDateString().split(' ').slice(1).join(' ').replace(regex, ', ')
+
+    expect(vm.$el.querySelector('.ar-date').textContent).toContain('Sep 18, 2020')
```

Comment:
Does this test pass today (Sep 24)?

To fix this, instead of doing `new Date()`, you can create a specified date and then check that the element contains it.

Reply:
Certainly, that change could be made! The original idea behind the unit test was to also include the formatting of the new date. It can be certainly be mocked and validated.